### ISSUE TITLE
Bug in invoice.go: `json:"paymentIntent"` (camel case instead of snake case)

### ISSUE THREAD

**Issue by nsf** [stripe/stripe-go#856]
Looks like a bug on this line:

https://github.com/stripe/stripe-go/blob/24c6ea237b6270ac71be612718a495e3a2dff740/invoice.go#L229

Should use snake case instead of camel case, e.g.:
```
PaymentIntent                *PaymentIntent           `json:"payment_intent"`
```

As in API: https://stripe.com/docs/api/invoices/object#invoice_object-payment_intent

**Comment by brandur-stripe** [comment]
Oops. Indeed it should! Fixing now.

**Comment by brandur-stripe** [comment]
We just shipped a fix version in 60.13.2. Thanks for the heads up @nsf!
